README.md

```diff
@@ -15,9 +15,84 @@ This repository houses on-chain smart contracts, Haskell off-chain interaction l
 - [`geniusyield-server-lib`](./geniusyield-server-lib/) serves endpoints using our off-chain code to easily interact with GeniusYield DEX in language of user's choice.
 - [`geniusyield-orderbot-lib`](./geniusyield-orderbot-lib/) cater to additional requirements such as building up of an order-book, receiving price feed, etc.
 
-## Spinning up server
+# Bot API Server
```

Comment:
Shouldn't it have `##` in prefix? Current way would also break table of contents.
Calling it "Bot ..." might not be a good idea as server could be spin up without having any signing key configured, say for a scenario of browser wallet where this API is still useful.

Reply:
I can add an extra level

Reply:
I have fixed the table of contents:

![image](https://github.com/geniusyield/dex-contracts-api/assets/2914096/a8a69f9a-21b7-4e9e-9dc8-d80b55ed9d90)